tests/qchem/test_structure.py

```diff
@@ -77,40 +78,184 @@ def test_reading_xyz_file(tmpdir):
         "delta_sz",
         "singles_exp",
         "doubles_exp",
+        "fermionic_singles_exp",
+        "fermionic_doubles_exp",
     ),
     [
-        (1, 5, 0, [[0, 2], [0, 4]], []),
-        (1, 5, 1, [], []),
-        (1, 5, -1, [[0, 1], [0, 3]], []),
-        (2, 5, 0, [[0, 2], [0, 4], [1, 3]], [[0, 1, 2, 3], [0, 1, 3, 4]]),
-        (2, 5, 1, [[1, 2], [1, 4]], [[0, 1, 2, 4]]),
-        (2, 5, -1, [[0, 3]], []),
-        (2, 5, 2, [], []),
-        (3, 6, 1, [[1, 4]], []),
+        (
```

Comment:
Is there a way to make this more compact or black always expands?

Reply:
I got it a bit more compact, but I think black will always expand some parts of it.